libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayEventSimulator.kt

```diff
@@ -120,6 +129,9 @@ internal class ReplayEventSimulator(
         // The frequency that replay updates will be broad-casted
         private const val replayUpdateSpeedMillis = 100L
```

Comment:
```suggestion
        private const val REPLAY_UPDATE_SPEED_MILLIS = 100L
```

Reply:
this commit is not yelling i swear
https://github.com/mapbox/mapbox-navigation-android/pull/2885/commits/b6d0b74b093aa6b04a59b708c3390639f74007e2